android/app/build.gradle

```diff
@@ -194,7 +194,7 @@ android {
         applicationId "me.rainbow"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 67
+        versionCode 72
```

Comment:
73 is live in the PlayStore so you need to bump it otherwise it will fail

Reply:
![Screen Shot 2022-06-08 at 11 58 30 AM](https://user-images.githubusercontent.com/296775/172663235-c395ceae-3943-40cb-aaa1-52d21fe89e23.png)

Bumped it to 75 otherwise the upload would fail anyway. But looking into the equivalent of `latest_testflight_build_number` for google play so we don't need to rely on this number